openupgrade_scripts/scripts/payment/15.0.2.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,183 @@
+---Models in module 'payment'---
+new model payment.refund.wizard [transient]
+# NOTHING TO DO: new model: https://github.com/odoo/odoo/commit/e0233a1010618450cf53b997c50964bed2bb1b1a
+
+---Fields in module 'payment'---
+payment      / account.payment          / source_payment_id (many2one)  : NEW relation: account.payment, isrelated: related, stored
+# NOTHING TO DO: it's related to source_transaction_id and the new refund feature
+
+payment      / account.payment.method.line / payment_acquirer_id (many2one): NEW relation: payment.acquirer, isfunction: function, stored
+# NOTHING TO DO
```

Comment:
You have to create records of this type for storing the journals of the `payment.acquirer`, following the inverse method:

https://github.com/odoo/odoo/blob/cdaca6d49281b6106db646ddfda2dd7e0d7c8388/addons/payment/models/payment_acquirer.py#L182